ModelBundle/Document/ContentType.php

```diff
@@ -211,7 +211,15 @@ public function setDefaultListable(array $defaultListable)
      */
     public function getName($language = 'en')
     {
-        return $this->names->get($language)->getValue();
+        if ($this->names->containsKey($language)) {
+            return $this->names->get($language)->getValue();
+        }
+
+        if ($this->names->count() > 0) {
+            return $this->names->first()->getValue();
```

Comment:
Should we return the first language or message saying the anme is not translated ?